CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# 2.1.0
+
+### Bug Fixes
+
+- [#17](https://github.com/okta/okta-vue/pull/17) Removes `onSessionExpired` behavior.
```

Comment:
Clarify that we're removing the _default_ behavior, and not the existence of the callback?  People are reading the CHANGELOG to decide how this impacts their code, and the above doesn't make that clear.